reference/pipeline/stages/index.md

```diff
@@ -80,31 +81,35 @@ to disable the Server Group which is newest, oldest, or previous
 (second-most-recently deployed) when this stage starts.
 
 ### Enable Server Group
-Enable a Server Group, which means that the Server Group will will resume
-handling traffic.
+Enable a Server Group, which means that the Server Group resumes handling
+traffic. The configuration of your Load Balancer determines how traffic is
+routed between newly-enabled Server Groups and any existing Server Groups.
```

Comment:
* If possible, start the definition with something other than a verbatim repeat of the name ;)

* Is it always "resumes"? Could this be a new server group being enabled, in which case this "begins handling of server traffic."? "Begins or resumes..." ?

* Re the load balancer, are there *two or more* server groups, or just two. If there can be more than two, use "among" rather than between.

Reply:
Yep, it always resumes handling traffic (I checked). Server Groups start handling traffic when they're created (when an image is deployed to them), so you would only need to explicitly enable a Server Group after it had already been disabled.